src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts

```diff
@@ -1153,22 +1153,25 @@ export class NotebookEditorWidget extends Disposable implements INotebookEditorD
 		const endTime = Date.now() + deadline.timeRemaining();
 
 		const execute = () => {
-			this._backgroundMarkdownRenderRunning = false;
-			if (this._isDisposed) {
-				return;
-			}
+			try {
+				this._backgroundMarkdownRenderRunning = true;
```

Comment:
Fixing this flag so that `_backgroundMarkdownRendering` doesn't bail on the second time through.